ui/src/components/nested-fields/QasNestedFields.vue

```diff
@@ -270,7 +275,13 @@ export default {
 
   methods: {
     add (row = {}) {
-      const payload = { ...this.rowObject, ...row }
+      const clonedRow = { ...row }
+
+      if (row[this.identifierItemKey]) {
+        delete clonedRow[this.identifierItemKey]
+      }
+
+      const payload = { ...this.rowObject, ...clonedRow }
```

Comment:
acho q poderia passar direto não ? `this.nested.push({ ...this.rowObject, ...clonedRow })`

Reply:
Poderia, porém, assim com payload fica mais claro do que está sendo feito

Reply:
como estavamos conversando no discord, não teria q usar o `extend` por conta de profundidade de objeto?

Reply:
Substituí